app/models/card.rb

```diff
@@ -8,20 +8,31 @@ class Card < ActiveRecord::Base
 
   validate :original_and_translated_are_different
 
-  def original_and_translated_are_different
-    if normalize(original_text) == normalize(translated_text)
-      errors.add(I18n.t("card.original"), I18n.t("card.original_translated"))
-      errors.add(I18n.t("card.translated"), I18n.t("card.translated_original"))
-    end
+  scope :ready_to_review, -> { where('review_date <= ?', Time.zone.now) }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.